specification/app/resource-manager/Microsoft.App/preview/2022-06-01-preview/ContainerApps.json

```diff
@@ -641,6 +641,16 @@
         "allowInsecure": {
           "description": "Bool indicating if HTTP connections to is allowed. If set to false HTTP connections are automatically redirected to HTTPS connections",
           "type": "boolean"
+        },
+        "ipSecurityRestrictionRules": {
```

Comment:
>ipSecurityRestrictionRules [](http://example.com/codeflow?start=9&length=26)

Is this name consistent with other RPs? Please check other RPs such as app service, SQL, etc to see what names they use.

Reply:
Renamed to maintain consistency with other services
![MicrosoftTeams-image](https://user-images.githubusercontent.com/41846928/180569586-fd4c03b4-b68b-4066-b52d-9fae2f37e318.png)